joshuaCarpentier/assignments/slidingwindow/lc209/lc209.js

```diff
@@ -23,21 +26,21 @@ console.log(
 
 // Solution
 // -----
-// let windowSum = 0,
-// minLength = Infinity,
-// windowStart = 0;
+let windowSum = 0,
+minLength = Infinity,
+windowStart = 0;
 
-// for (windowEnd = 0; windowEnd < arr.length; windowEnd++) {
-// windowSum += arr[windowEnd]; // add the next element
-// // shrink the window as small as possible until the 'window_sum' is smaller than 's'
-// while (windowSum >= s) {
-//   minLength = Math.min(minLength, windowEnd - windowStart + 1);
-//   windowSum -= arr[windowStart];
-//   windowStart += 1;
-// }
-// }
+for (windowEnd = 0; windowEnd < arr.length; windowEnd++) {
```

Comment:
It seems like your code is not written inside a function. Can you please make sure you contain your code inside a function and test it before submitting it?